### ISSUE TITLE
The amount reflected to User while picking services should be Total + VAT from start

### ISSUE THREAD

**Issue by rgnarok** [atul-gautam-au6/carwazi-issues#238]

**Comment by rgnarok** [comment]
![image](https://github.com/atul-gautam-au6/carwazi-issues/assets/85341053/ff16e159-7246-4bca-a424-d1cacd449480)


![image](https://github.com/atul-gautam-au6/carwazi-issues/assets/85341053/1495f358-baff-449e-a362-33eb670ed892)


@Ahlam000  it should be 2300 not 2000 on the listing screen as mentioned from the start it must be cost + VAT

**Comment by Ahlam000** [comment]
@rgnarok  please check at the bottom i have put cost with vat only i,e is from start in the list we show the cost only of the item as have to show the actual cost at the payment page also if vat should be from the start then price should come from backend which add vat also in the backend @atul-gautam-au6  please check if it is possible

**Comment by rgnarok** [comment]
@Ahlam000  this is not correct as the amount to be shown while picking up services need to be cost +VAT and picking the service comes on this screen
![image](https://github.com/atul-gautam-au6/carwazi-issues/assets/85341053/76c26f3d-237e-4885-8ded-59e4c9a65c86)


So please fix this
